healthcare/api-client/v1/dicom/dicom_stores.py

```diff
@@ -0,0 +1,455 @@
+# Copyright 2018 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import json
+import os
+
+from googleapiclient import discovery
+
+
+# [START healthcare_get_client]
+def get_client():
+    """Returns an authorized API client by discovering the Healthcare API and
+    creating a service object using the service account credentials in the
+    GOOGLE_APPLICATION_CREDENTIALS environment variable."""
+    api_version = "v1"
+    service_name = "healthcare"
+
+    return discovery.build(service_name, api_version)
+
```

Comment:
Nit: can you remove 2 empty lines after the `return` statement and `[END healthcare_get_client]`?
Also here and there.

Reply:
I would but that's an effect of running the black tool that we're supposed to be using to lint everything. I'd have to manually go back and remove the lines every time I run the lint tool.

Reply:
@kurtisvg Is there a way to instruct black not inserting the empty lines?

Reply:
If you indent the comment to match the function line, Black will likely treat it as part of the function and won't add the extra lines (otherwise I believe it's a PEP8 rule to have 2 empty lines after a top level function).

Reply:
That's not ideal because that goes against style for writing comments, no?